Modifications performed by ValdikSS (iam@valdikss.org.ru). I
created this patch from his update to show the changes.

diff -Naur rtl8192ce_linux_2.6.0006.0321.2011/HAL/rtl8192/Makefile rtl8192ce_linux_2.6.0006.0321.2011-new/HAL/rtl8192/Makefile
--- rtl8192ce_linux_2.6.0006.0321.2011/HAL/rtl8192/Makefile	2011-02-11 02:37:42.000000000 -0600
+++ rtl8192ce_linux_2.6.0006.0321.2011-new/HAL/rtl8192/Makefile	2011-08-03 10:30:32.000000000 -0500
@@ -4,7 +4,7 @@
 KVER  := $(shell uname -r)
 MODDESTDIR := /lib/modules/$(KVER)/kernel/drivers/net/wireless/
 
-ifeq ($(shell uname -r|cut -d. -f1,2), 2.6)
+ifeq ($(shell uname -r|cut -d- -f1), 3.0)
 ###################### KERNEL 2.6 ################################
 ifeq ($(NIC_SELECT),RTL8192CE)
 sub_folder = rtl8192c
@@ -739,7 +739,7 @@
 ###################### KERNEL 2.4 ################################
 LD := ld
 KSRC := /lib/modules/$(KVER)/build
-CONFIG_FILE := $(KSRC)/include/linux/autoconf.h
+CONFIG_FILE := $(KSRC)/include/generated/autoconf.h
 CFLAGS += -DLINUX -D__KERNEL__ -DMODULE -O2 -pipe -Wall
 CFLAGS += -I$(KSRC)/include -I.
 CFLAGS += -DMODVERSIONS -DEXPORT_SYMTAB -include $(KSRC)/include/linux/modversions.h
